website/Gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+
+gem 'middleman', '~> 4.2'
+gem 'middleman-autoprefixer', '~> 2.7'
+gem 'tzinfo-data', platforms: [:mswin, :mingw, :jruby]
```

Comment:
`ruby` is the only supported platform at this point. Is this necessary?

Reply:
heh copypasta from middleman tutorial